yaml/service.go

```diff
@@ -61,6 +64,32 @@ func (s *ServiceSlice) UnmarshalYAML(unmarshal func(interface{}) error) error {
 		return err
 	}
 
+	// iterate through each element in the service slice
+	for _, service := range *serviceSlice {
+		// implicitly set `pull` field if empty
+		if len(service.Pull) == 0 {
+			service.Pull = constants.PullNotPresent
+		}
```

Comment:
Set a default pull policy of `not_present` for the `github.com/go-vela/types/yaml.Service` type